synapse/storage/databases/main/search.py

```diff
@@ -101,20 +101,20 @@ def store_search_entries_txn(
             txn.execute_batch(sql, args1)
 
         elif isinstance(self.database_engine, Sqlite3Engine):
-            sql = (
-                "INSERT INTO event_search (event_id, room_id, key, value)"
-                " VALUES (?,?,?,?)"
-            )
-            args2 = (
-                (
-                    entry.event_id,
-                    entry.room_id,
-                    entry.key,
-                    _clean_value_for_search(entry.value),
-                )
-                for entry in entries
+            self.db_pool.simple_insert_many_txn(
```

Comment:
I'm not sure if it is clearer switching this to `simple_insert_many_txn`, but the code is identical... (I think).